packages/plugin-core/src/test/testUtilsV3.ts

```diff
@@ -360,9 +360,22 @@ export function setupBeforeAfter(
 
     // workspace has not upgraded
     if (!opts?.noSetInstallStatus) {
-      sinon
-        .stub(VSCodeUtils, "getInstallStatusForExtension")
-        .returns(InstallStatus.NO_CHANGE);
+      // try to remove any existing stub in case it exists
+      // this is because we have tests that call `setupBeforeAfter` as well as
+      // in describeMultiWS > [[../packages/plugin-core/src/test/testUtilsV3.ts#^lk3whwd4kh4k]]
+      // TODO: keep in place until we completely remove `setupBeforeAndAfter`
+      try {
+        // @ts-ignore
+        sinon
+          .stub(VSCodeUtils, "getInstallStatusForExtension")
+          .returns(InstallStatus.NO_CHANGE);
+      } catch (e) {
+        // eat it.
+        sinon.restore();
```

Comment:
This restores everything, so it could restore more than intended. It's temporary, but would it be safer to just skip stubbing it if the first stub failed instead?

Reply:
it would - unfortunately this still causes tests to fail since there are other sinon stubs underneath that will still throw an error, as well as potential stubs in the `beforeHook`. decided it wasn't worth the trouble